package.json

```diff
@@ -121,6 +121,10 @@
     "ocaml-language-server": "1.0.12",
     "oni-neovim-binaries": "^0.0.7",
     "oni-ripgrep": "0.0.3",
+    "shell-env": "^0.3.0",
```

Comment:
Could you please set the version numbers to fixed versions? (IE, `0.3.0` instead of `^0.3.0`).

Sometimes we get burned by dependencies updating and breaking us.